pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -15,6 +16,14 @@ def single_level_multiindex():
                       codes=[[0, 1, 2, 3]], names=['first'])
 
 
+@pytest.fixture
+def frame_random_data_integer_multi_index():
```

Comment:
Is this fixture duplicative of the one in conftest?

Reply:
@WillAyd : probably yes.. it is a dataframe with random data with an integer multi-level index, so `multiindex_year_month_day_dataframe_random_data` fixture could probably be used instead.

Reply:
OK - if you want to tackle that in a follow up always good to reduce duplication